contrib/configure-devel

```diff
@@ -14,7 +14,6 @@ basedir=$(cd $(dirname $0) && pwd)
 $basedir/../configure \
 	--enable-gtest \
 	--enable-examples \
-	--with-valgrind \
```

Comment:
pls keep it. gtest can do --without-valgrind

Reply:
Done.